### DISCUSSION TITLE
请问现在是不是不能在通过cname方式将子域名接入cloudflare了？

### DISCUSSION THREAD

**Discussion by nj-dream** [XIU2/CloudflareSpeedTest#336] [讨论]
我当前的域名用dnspod，大部分用户在国内。现在想用一个子域名为国外的朋友提供服务，我在cloudflare添加子域名时提示不允许。
查了一下，以前似乎可以通过partner的方式，以cname添加子域名，但现在好像不行了。请问大家，现在还有什么方式能将子域名接入cloudflare呢？

**Comment by cxw620** [comment]
> 是的，暂时没有办法

如果在CF上面添加域名, 但只通过NS授权子域名给CF解析可以吗? 会不会一段时间后CF会自动清除没有解析NS的域名.

**Comment by XIU2** [comment]
@cxw620 Cloudflare 会定期检查 NS，发现不对就会立即停用。
